packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -133,7 +132,7 @@ const cfg = (options?: Options): Config => {
           "button-small": defaultTokens.formBoxSmallHeight,
           "button-normal": defaultTokens.formBoxNormalHeight,
           "button-large": defaultTokens.formBoxLargeHeight,
-          checkbox: defaultTokens.iconSmallSize,
+          checkbox: defaultTokens.iconMediumSize,
```

Comment:
Is this change expected? From small to medium

Reply:
removed completly as it's wrong to use this, should be used another class, eg lineHeightNormal